testing/Scripts/diff-remove-abspath

```diff
@@ -0,0 +1,13 @@
+#! /usr/bin/env bash
+#
+# Replace absolute paths with the basename.
+
+if [ $(uname) == "Linux" ]; then
```

Comment:
This looks like it might have been added from elsewhere.

For new code, my suggestion would be to clean this up somewhat, e.g.,

```suggestion
#!/usr/bin/env bash
#
# Replace absolute paths with the basename.

if [ "$(uname)" == "Linux" ]; then
```